ibis/tests/util.py

```diff
@@ -125,6 +127,7 @@ def tearDown(self):
         for t in self.temp_views:
             self.con.drop_view(t, force=True)
 
+#        self.con.drop_udf(db=self.test_data_db, force=True)
```

Comment:
this will get replaced with calls to `drop_udf` that remove all functions in `temp_functions`